### DISCUSSION TITLE
How do I make Numerical Range filter type work on percentage values?

### DISCUSSION THREAD

**Discussion by kyutums** [apache/superset#20916] [Q&A / Help]
I am trying to make a filter for percentages. What is displayed for the value in a table is %, but actual value in database is between 0 and 1. Example, .45 in database is being displayed as 45.00% in the Chart (I set this in D3 Format of Chart). However, when I choose "Numerical range" in Filter Type of dashboard that shows the chart, the only numbers in the range are integers (0 and 1). This results in - basically - no way to move the filter for the numbers between 0 and 1 (ex. .45 pr 45%). Any tips on how to filter percentages?

This is how the filter looks like: [https://imgur.com/a/bgvnN3C](url)

**Comment by HiepPHX** [comment]
Super late to the party, but you can work around this by adding a column in your Dataset SQL to simulate the percentage range. For example:

SELECT *, percentage_value * 100 as percentage_value_for_filter
FROM your_table

And then you set your filter to point to the percentage_value_for_filter. Now you can properly slide between 0-45 instead of 0-1.
